searx/engines/soundcloud.py

```diff
@@ -1,102 +1,146 @@
 # SPDX-License-Identifier: AGPL-3.0-or-later
-"""
- Soundcloud (Music)
-"""
+"""SoundCloud is a German audio streaming service."""
 
 import re
-from json import loads
 from urllib.parse import quote_plus, urlencode
-from lxml import html
+import datetime
+
 from dateutil import parser
+from lxml import html
+
 from searx.network import get as http_get
 
-# about
 about = {
-    "website": 'https://soundcloud.com',
-    "wikidata_id": 'Q568769',
-    "official_api_documentation": 'https://developers.soundcloud.com/',
-    "use_official_api": True,
+    "website": "ttps://soundcloud.com",
```

Comment:
`"ttps://soundcloud.com",` --> `"https://soundcloud.com",`

Reply:
> `"ttps://soundcloud.com",` --> `"https://soundcloud.com",`

Oops .. fixed in f45d4145e